### DISCUSSION TITLE
Problem with Album Art Post 20230630 Update

### DISCUSSION THREAD

**Discussion by ccaper** [rern/rAudio#1379] [Bug Reports]
Hi, relatively new rAudio user (4 weeks, love it), and first post here.

Setup:
RPi 4 8GB
RPi is headless, and nothing plugged into it
Music files are on a USB thumb drive that is mounted on my router, VFAT32 filesystem, smb mounted to rAudio.

Before taking the 20230630 update, I would copy new files to the USB drive, go to rAudio, update library (usually new files only, sometimes I would rebuild entire database), then go into Library Albums, and update thumbnails.  All this worked fine in the past.

Friday June 30th, I took the latest update (20230630), and I was already on the previous latest update.  After taking the update, I added some new music files, then updated library (new files), then update thumbnails.  Update thumbnails reported that some albums "No coverart found.".  It reported this error for both some new albums I had added, and some existing albums that had been added BEFORE the update, that currently had album art showing, and had the rAudio cover.jpg, coverart.jpp, and thumb.jpg files existing in the directory.

I then tried a rebuild complete database, and update thumbnails, same issue on same albums.

I then unmounted my thumb drive, deleted all files off the thumb drive, recopied them, remounted thumb drive on router, rebuilt entire rAudio database, update thumbnails, same problem, same albums.  I then double checked each album to ensure each song had an embedded image, they did.  I then took one of the problematic albums, re-embedded the image, copied it back to the USB thumb drive, updated library, update thumbnails, same issue.  And this was on an album that was properly displaying album art in rAudio before the 20230630 update (and I had added new music of the type described below that is now giving me problems in the previous release had had no issues).  So I'm pretty sure this issue is related to the 20230630 update, and not a problem with the music files (meaning it wasn't one of the new songs I had tried adding after taking the 20230630 update).

What's interesting is the album art problem seems to be related to a particular type of song I'm adding.  All my files are flac, but this seems to only be related to albums that are Various Artists.  My various artists songs filenames are of the format 'VA - <album-name> -<track-number>- <artist> - <song-title>.flac', for example 'VA - 100% Dynamite! -01- Willie Williams - Armageddon Time.flac', where the album name is "100% Dynamite!", track number is "01", artist is "Willie Williams", and song title is "Armageddon Time".  My flac tags have the album artist as "VA" for each song on the various artists album, and the artist as the artist performing that song on the various artists album.

Furthermore, when I go into the library album view for the album that it can not find coverart, all I see is the album name, no track listings on that album, no default image.

Again, this seems isolated to the latest release.  And this is happening for both music files I added BEFORE the update, and after the update, and seems to be related to only various artists.

For a non various artists album, I deleted the cover, coverart, and thumb, updated thumbs, and it found the coverart just fine.

Any ideas on how to fix? Is this a bug with the 20230630 release?  Is it possible to roll back my rAduio to previous release where this worked?

Thank you, hope I proved enough info to help with this possible bug.

**Comment by GDV0** [comment]
Hi ccaper,

I had exactly the same issue with coverarts in library album view.
I finally fix it by discovering that if the album folder name contains 2 consecutive spaces, then coverart is not displayed.

Hope this could help
